simul/platform/cliutils.go

```diff
@@ -56,6 +56,7 @@ func SSHRun(username, host, command string) ([]byte, error) {
 		addr = username + "@" + addr
 	}
 
+	log.Lvl4("Going to ssh to", addr, command)
```

Comment:
```suggestion
	log.Lvl2("Going to ssh to", addr, command)
```
I would put this even in Lvl2, seems quite interesting ;)